packages/fxa-auth-db-mysql/lib/db/schema/patch-112-113.sql

```diff
@@ -0,0 +1,10 @@
+-- -- create userCustomers table for storing fxa user to stripe customer relationship
+
+
+CREATE TABLE IF NOT EXISTS userCustomers (
+  fxaUserId VARCHAR(255) PRIMARY KEY,
```

Comment:
I think `createdAt` is a good idea. Though, `fxaUserId` should probably instead be `uid binary(16) NOT NULL PRIMARY KEY` for consistency & maybe joins with the `accounts` table. I don't remember why we use a `binary(16)` column here, though. Storage size maybe?

```
✗ mysql --host 127.0.0.1 -u root fxa -e 'show create table accounts'
...
CREATE TABLE `accounts` (
  `uid` binary(16) NOT NULL,
  `normalizedEmail` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
  `email` varchar(255) COLLATE utf8_unicode_ci NOT NULL,
  `emailCode` binary(16) NOT NULL,
  `emailVerified` tinyint(1) NOT NULL DEFAULT '0',
  `kA` binary(32) NOT NULL,
  `wrapWrapKb` binary(32) NOT NULL,
  `authSalt` binary(32) NOT NULL,
  `verifyHash` binary(32) NOT NULL,
  `verifierVersion` tinyint(3) unsigned NOT NULL,
  `verifierSetAt` bigint(20) unsigned NOT NULL,
  `createdAt` bigint(20) unsigned NOT NULL,
  `locale` varchar(255) COLLATE utf8_unicode_ci DEFAULT NULL,
  `lockedAt` bigint(20) unsigned DEFAULT NULL,
  `profileChangedAt` bigint(20) unsigned DEFAULT NULL,
  `keysChangedAt` bigint(20) unsigned DEFAULT NULL,
  `ecosystemAnonId` text CHARACTER SET ascii COLLATE ascii_bin,
  PRIMARY KEY (`uid`),
  UNIQUE KEY `normalizedEmail` (`normalizedEmail`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci |`
```